usefulUtilities/userLocation/userLocation.js

```diff
@@ -0,0 +1,208 @@
+//
+//  userLocation.js
+//
+//  Created by Zach Fox on 2019-04-02
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    var request = Script.require("./modules/request.js").request;
```

Comment:
Is there a reason why we can't access request in client/server?  I think we are adding a lot of the same module over and over again.  Lol do a search for request in the hifi-content code base.

Reply:
discussed in Slack!